.stylelintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "extends": [ "stylelint-config-recommended", "stylelint-config-sass-guidelines" ],
+    "extends": [ "stylelint-config-recommended", "stylelint-config-sass-guidelines", "stylelint-config-css-modules" ],
```

Comment:
Any chance we can make this multi-line and get cleaner diffs? May be something we'll have to push down the line, and use a `.js` file so we can have trailing commas.